Workbooks/WorkbooksMetadata.json

```diff
@@ -1495,6 +1495,20 @@
         "subtitle": "",
         "provider": "Azure Sentinel Community",
         "featureFlag": "DSTIMWorkbook"
+      },
+      {
+        "workbookKey": "Log4jPostCompromiseHunting",
+        "logoFileName": "",
+        "description": "This hunting workbook is intended to help identify activity related to the Log4j compromise discovered in December 2021.",
+        "dataTypesDependencies": [ "SecurityNestedRecommendation", "AzureDiagnostics", "OfficeActivity", "W3CIISLog", "AWSCloudTrail", "SigninLogs", "AADNonInteractiveUserSignInLogs", "imWebSessions", "imNetworkSession"],
+        "dataConnectorsDependencies": [],
+        "previewImagesFileNames": [ "Log4jPostCompromiseHuntingBlack.png", "Log4jPostCompromiseHuntingWhite.png" ],
+        "version": "1.0",
+        "title": "Log4j Post Compromise Hunting",
+        "templateRelativePath": "Log4jPostCompromiseHunting.json",
+        "subtitle": "",
+        "provider": "Azure Sentinel Community",
+        "featureFlag": "Log4jPostCompromiseHunting"
```

Comment:
@samikroy, do you want this workbook to be available behind a feature flag? Just FYI... by adding this feature flag it won't be publicly available... Thanks!!

Reply:
Thank you for the suggestion @v-rucdu. The idea for the workbook was to 
1. Compliment the analytics & hunting rules on a snapshot view. 
2. Eventually become a part of the solution. 
https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/Apache%20Log4j%20Vulnerability%20Detection

Please let me know your suggestions if the workbook needs more additions.

Reply:
@samikroy : If you want the workbook to be part of the solution, than in that case it should be placed inside the respective solutions folder. You can refer [ApacheHttpServer](https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/ApacheHTTPServer) solution for understanding the folder structure.

Thanks!!!

Reply:
@v-rucdu  - Would love to have this workbook in both the places.
1. Generally available in workbooks template gallery 
2. As a part of the solution.
Please let me know for additional changes.

Reply:
@v-rucdu - Have looked into the shared solution [ApacheHTTPServer](https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/ApacheHTTPServer) but the [Apache Log4j Vulnerability Detection](https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/Apache%20Log4j%20Vulnerability%20Detection) structure is different so in case we create a folder named workbook inside the solution and then how to link the workbook inside the main template 
https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Apache%20Log4j%20Vulnerability%20Detection/Package/mainTemplate.json

Reply:
@samikroy : You just add the workbook to the solution --> workbook folder we will take care of creating the package and publishing it in market place. Incase you want to understand how the package is created please refer the documentation [here](https://github.com/Azure/Azure-Sentinel/blob/master/Tools/Create-Azure-Sentinel-Solution/README.md).

Also, if you want to have public access to the workbook post deployment, please remove the feature flag attribute from WorkbookMetadata.json

Thanks!!!

Reply:
@v-rucdu - Thank you for clarifying. Have done the changes in #3888 
And updated the current PR to make the workbook public. 
Please approve.